examples/nl-core-BowelFunction-02-DeviceUseStatement-01.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<DeviceUseStatement xmlns="http://hl7.org/fhir">
+   <id value="nl-core-BowelFunction-02-DeviceUseStatement-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-MedicalDevice"/>
+   </meta>
+   <status value="active"/>
+   <subject>
+      <reference value="Patient/nl-core-Patient-01"/>
+      <type value="Patient"/>
+      <display value="Patient, Johanna Petronella Maria (Jo) van Putten-van der Giessen"/>
+   </subject>
+   <device>
+      <reference value="Device/nl-core-BowelFunction-02-Device-01"/>
+      <type value="Device"/>
+      <display value="Elastisch broekje"/>
+   </device>
+   <reasonReference>
+      <reference value="Observation/nl-core-BowelFunction-02"/>
+      <type value="Observation"/>
+      <display value="Bowel function observation, fecal continence: incidenteel ontlastingsverlies, defecation consistency: Zachte stukjes met gehavende randen"/>
+   </reasonReference>
```

Comment:
The validator gives the following error:

```
  -  error at DeviceUseStatement.reasonReference[0] (18, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-MedicalDevice path DeviceUseStatement.reasonReference[0] (fhirPath = true and (resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-BowelFunction') or resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-FunctionalOrMentalStatus') or resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Mobility') or resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Stoma'))): The URL 'Observation/nl-core-BowelFunction-02' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at DeviceUseStatement.reasonReference[0] (18, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-MedicalDevice path DeviceUseStatement.reasonReference[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Problem')): The URL 'Observation/nl-core-BowelFunction-02' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
```

Reply:
Which validator did you use? In this PR these errors are not present.

Reply:
QA tool in the repo and simplifier.